examples/dbc2val/dbcreader.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+########################################################################
+# Copyright (c) 2020 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+########################################################################
+
+import can, cantools
```

Comment:
`python-cantools` v 34.0.0  (MIT licensed) is approved as workswith, but python-can (LGPL) is not.

Reply:
Ayay always 20% coding 80% red tape at Eclipse :) Thanks for pointing it out. I think it can be done: By definition this whole thing is an optional example, and not part of the server, so workswith should work. For python-can it is just one of those standard packages where verison doesn't really matter. If there isn't any CQ to piggy back, can we do


Python Can 3.3.3 for example here https://github.com/hardbyte/python-can/tree/3.3.3 with LGPL license or alternatively here https://pypi.org/project/python-can/3.3.3/ (dame thing) 

I understand LGPL is ok for workswith?

Reply:
The CQ for python-can is filed, but as of https://www.eclipse.org/lists/eclipse.org-committers/msg01241.html , I feel confident with already merging this PR, since python-can is not a core part of Eclipse Kuksa, we do not distribute it in our repo, and its use is just intended for an example.